Automattic/woocommerce-payments
# feat: add general settings section

Fixes https://github.com/Automattic/woocommerce-payments/issues/1634

#### Changes proposed in this Pull Request









Adds the "General Settings" section to the settings.

Waiting for https://github.com/Automattic/woocommerce-payments/pull/1690 to get merged before implementing [these sections](https://d.pr/i/P6cE68) on the description (since it touches the same files).

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Section is displayed
- Input change also changes the help text

![2021-04-28 09 53 11](https://user-images.githubusercontent.com/273592/116425012-99975000-a807-11eb-9c6b-22a7325ad873.gif)

![Screen Shot 2021-04-28 at 9 50 14 AM](https://user-images.githubusercontent.com/273592/116424485-2beb2400-a807-11eb-8159-839ed634f267.png)


-------------------
